integrations/gcp/.port/resources/blueprints.json

```diff
@@ -0,0 +1,169 @@
+[
+  {
+    "identifier": "gcpOrganization",
+    "title": "GCP Organization",
+    "icon": "GoogleCloud",
+    "schema": {
+      "properties": {},
+      "required": []
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "aggregationProperties": {},
+    "relations": {}
+  },
+  {
+    "identifier": "gcpProject",
+    "title": "GCP Project",
+    "icon": "GoogleCloud",
+    "schema": {
+      "properties": {}
+    },
+    "mirrorProperties": {},
+    "calculationProperties": {},
+    "aggregationProperties": {}
+  },
+  {
+    "identifier": "bucket",
+    "title": "Bucket",
+    "icon": "GoogleCloud",
+    "schema": {
+      "properties": {
+        "url": {
+          "title": "URL",
+          "format":"url",
+          "type": "string"
+        },
+        "name": {
+          "title": "Name",
+          "type": "string"
+        },
+        "owner": {
+          "title": "Owner",
+          "type": "string"
+        },
+        "location": {
+          "title": "Location",
+          "type": "string"
+        },
+        "objectRetention": {
+          "title": "Object Retention",
+          "type": "string"
+        }
+      }
+    },
```

Comment:
adjust the order of resource by the relations, project needs to be before bucket etc.. same for mapping